server/schedule/basic_cluster.go

```diff
@@ -49,6 +49,55 @@ type BasicCluster struct {
 	ReadStatistics  cache.Cache
 }
 
+// NewDiffMap creates a DiffMap
```

Comment:
Sentences should end with a period.

Reply:
I'm confused that sentences end without period in other places of this file, should I change them?

Reply:
sure.